<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>386</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>txt2img</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>use anti-aliasing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="genImgButton">
          <property name="text">
           <string>Generate image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Output image</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>width</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="imgYEdit">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="inputMask">
                 <string>0000; </string>
                </property>
                <property name="text">
                 <string>1024</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>height</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="imgXEdit">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="inputMask">
                 <string>0000; </string>
                </property>
                <property name="text">
                 <string>768</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="imgSizeSetButton">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>set</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="widgetResizable">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>100</width>
                 <height>100</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>folder:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="folderLabel_">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLineEdit" name="outFilenameLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>151</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>type in box &amp; image file name</string>
           </property>
          </widget>
          <widget class="QPushButton" name="selectFolderButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>..</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>save box  + image</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="fontSizeComboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="letterSpacingSpinBox">
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_copy"/>
   <addaction name="action_cut"/>
   <addaction name="action_paste"/>
   <addaction name="separator"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_font"/>
   <addaction name="separator"/>
   <addaction name="action_fontSize"/>
   <addaction name="separator"/>
   <addaction name="action_letterSpacing"/>
   <addaction name="separator"/>
   <addaction name="action_bold"/>
   <addaction name="action_italic"/>
   <addaction name="action_underline"/>
   <addaction name="separator"/>
   <addaction name="action_justify"/>
   <addaction name="action_center"/>
   <addaction name="action_left"/>
   <addaction name="action_right"/>
   <addaction name="separator"/>
   <addaction name="action_zoomin"/>
   <addaction name="action_zoomout"/>
   <addaction name="action_nozoom"/>
  </widget>
  <action name="action_bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textbold.png</normaloff>:/:/images/textbold.png</iconset>
   </property>
   <property name="text">
    <string>&amp;bold</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textitalic.png</normaloff>:/:/images/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>&amp;italic</string>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textunder.png</normaloff>:/:/images/textunder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;underline</string>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
  </action>
  <action name="action_fontSize">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Font size</string>
   </property>
  </action>
  <action name="action_justify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textjustify.png</normaloff>:/:/images/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>&amp;justify</string>
   </property>
   <property name="toolTip">
    <string>Justify</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="action_center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textcenter.png</normaloff>:/:/images/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>c&amp;enter</string>
   </property>
   <property name="toolTip">
    <string>Align Center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textright.png</normaloff>:/:/images/textright.png</iconset>
   </property>
   <property name="text">
    <string>&amp;right</string>
   </property>
   <property name="toolTip">
    <string>Align right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/textleft.png</normaloff>:/:/images/textleft.png</iconset>
   </property>
   <property name="text">
    <string>&amp;left</string>
   </property>
   <property name="toolTip">
    <string>Align left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/editcopy.png</normaloff>:/:/images/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/editcut.png</normaloff>:/:/images/editcut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/editpaste.png</normaloff>:/:/images/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/editundo.png</normaloff>:/:/images/editundo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/editredo.png</normaloff>:/:/images/editredo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_zoomin">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/zoomin.png</normaloff>:/:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>&amp;zoomin</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=, Ctrl+=</string>
   </property>
  </action>
  <action name="action_zoomout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/zoomout.png</normaloff>:/:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>&amp;zoomout</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-, Ctrl+-</string>
   </property>
  </action>
  <action name="action_nozoom">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/nozoom.png</normaloff>:/:/images/nozoom.png</iconset>
   </property>
   <property name="text">
    <string>&amp;nozoom</string>
   </property>
   <property name="toolTip">
    <string>Cancel zoom</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/:/images/filesave.png</normaloff>:/:/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;save</string>
   </property>
   <property name="toolTip">
    <string>Save image and box</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_letterSpacing">
   <property name="text">
    <string> </string>
   </property>
   <property name="toolTip">
    <string>letter spacing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_bold</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textBold()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_italic</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textItalic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_underline</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>textUnderline()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSizeComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>textSize(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_zoomin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_zoomout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_nozoom</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancelZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_save</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>textBold()</slot>
  <slot>textItalic()</slot>
  <slot>textUnderline()</slot>
  <slot>textSize(QString)</slot>
  <slot>clipboardDataChanged()</slot>
  <slot>zoomin()</slot>
  <slot>zoomout()</slot>
  <slot>cancelZoom()</slot>
  <slot>save()</slot>
 </slots>
</ui>
